### DISCUSSION TITLE
Hotkey alt / option + L

### DISCUSSION THREAD

**Discussion by Vinczi** [ThijmenDam/BarTranslate#11] [Ideas]
Hey!
App is perfect and it's working great, although hardcoded hotkey "alt / option + L" is problematic for Polish users because we have been using it for our letter "Ł". 

In the app itself it isn't as problematic because we could just type "L" instead but since it is global hotkey it's not allowing to type "Ł" anywhere in the system.

Cheers!

**Comment by ThijmenDam** [comment]
Hey, thanks for the feedback! Great point 👍🏻 

I'll make the shortcut local instead of global as a quick fix! Of course it is better if the keys are configurable, but for now I unfortunately do not have the time to implement this.

**Comment by Vinczi** [comment]
Sure thing! I understand it's your side project and I really appreciate the idea behind it. I'd love to provide more help, but since I'm an ux designer I can only suggest some usability tweaks.

**Reply by ThijmenDam** [reply]
Hi @Vinczi, your feature request has been included in [BarTranslate 0.4.0](https://github.com/ThijmenDam/BarTranslate/releases/tag/0.4.0)! 🙂

Keyboard shortcuts are now localized (except for showing/hiding the app). On top of that they are now configurable.

**Reply by Vinczi** [reply]
Great Job @ThijmenDam! I'm testing it and it seems to work like a dream :)
